src/pages/content/grammar.mdx

```diff
@@ -1,12 +1,145 @@
 ---
 title: Grammar
-description: >
-  Lorem ipsum dolor sit amet, consectetur adipiscing
-  elit, sed do eiusmod tempor incididunt ut labore et
-  dolore magna aliqua.
 ---
 
-Coming soon
+## Acronyms and abbreviations
+
+Acronyms are like inside jokes. If you aren’t in the know they don’t make sense. Use acronyms to
+save space, but not at the risk of confusing people. Spell out an acronym on first use and include
+it in parentheses. ​
+
+_A service level agreement (SLA) is a contract between you and your customers._
+
+### Bold and italics
+
+Use bold to refer to UI elements. Avoid italics or quotation marks. Using bold is a bold statement.
+A little bit goes a long way, so use bold sparingly. ​ Don't get too bold in notifications. Only use
+bold for headings, and not in the description.
+
+<BestPractice>
+  <Do>
+    <p>
+      Click on <b>Settings</b> in Admin Center
+    </p>
+  </Do>
+  <Dont>
+    <p>Click on “Settings” in Admin Center</p>
+  </Dont>
+</BestPractice>
+
+## Capitalization
+
+Follow [standard capitalization
+rules](https://owl.purdue.edu/owl/general_writing/mechanics/help_with_capitals.html) for English.
+Capitalize names of people, companies, and product features and use lowercase for just about
+everything else. ​ At Zendesk, we use [sentence
+case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case) for headings, subheadings, and
+buttons.
+
+## Contractions
+
+Contractions are A-OK. They make sentences shorter and easier to read. Avoid them in flows that feel
+formal like billing, security settings, or account recovery.
+
+## Date and time
+
+Use unicode standards for date and time. Refer to your location-specific standards on
+[CLDR](https://zendesk.atlassian.net/wiki/spaces/globalization/pages/536936649/ICU+Intl+CLDR+at+Zendesk#ICU,Intl&CLDRatZendesk-WhyuseCLDR?).
+​ Zendesk uses relative time to show elapsed time. Avoid vague terms like “now,” “soon,” “later,” or
+“in a while.”
+
+| When the event happened                             | Example                   |
+| --------------------------------------------------- | ------------------------- |
+| Less than a minute ago today.                       | < 1 minute ago            |
+| More than a minute and less than an hour ago today. | 43 minutes ago 43 min ago |
```

Comment:
Example seems like a bug.

Reply:
I'm not sure what ya mean

Reply:
"43 minutes ago 43 min ago"

Reply:
Good catch, [looks like there's a break in the google doc.](https://share.getcloudapp.com/2Nu5PXXd) Will fix